configs/ray_tune_configs/ppo_config.yml

```diff
@@ -0,0 +1,69 @@
+tune_config:
+  mode: "max"
+  metric: "mean_reward"
+  search_alg: "bohb" # null
```

Comment:
maybe it's worth making random search option explicit instead of using `null` for it

Reply:
yeah makes sense. will do it.